ote/src/cljs/ote/views/transit_visualization/change_icons.cljs

```diff
@@ -72,6 +72,6 @@
 
     ;; Add route ending icon if route ending change is detected
     (when (str/includes? (str change-type) "removed")
-      [:div {:style {:width "10%"}}
+      [:div {:style {:width "10%"} :title "Mahdollisesti päättyvä reitti"}
```

Comment:
String used in many places, localized id would be nice :)

Reply:
Maybe next time